src/components/comments/Question.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+export default function Question(props) {
+  return <div style={styles.questionStyle}>{props.questionText}</div>;
+}
+
+const styles = {
+  questionStyle: {
+    backgroundColor: "#EEF0F1",
+    borderRadius: "25px",
+    textAlign: "left",
+    width: "80%",
+    padding: "10px",
+    fontFamily: "Mulish",
+    fontStyle: "normal",
+    fontWeight: "bold",
+    fontSize: "24px",
+    lineHeight: "30px",
+    alignItems: "left",
```

Comment:
Is there a reason you're setting this? I don't see any `flex` anywhere?

Reply:
oops forgot to delete that